apps-devstg/us-east-1/datalake--/README.md

```
@@ -4,6 +4,54 @@ This Terraform layer is used for creating resources that are required by the dat
+
+### 3. Data Science Account - Lakehouse Layer (Optional but Recommended)
+For complete data processing capabilities, also apply the lakehouse layer:
+```bash
+# In data-science account  
+cd data-science/us-east-1/lakehouse-demo--
+leverage tf apply
+```
+
+## Data Flow
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)